test/unit/platforms/os_detect_test.rb

```diff
@@ -148,6 +154,16 @@ def debian_scan(id, version)
       end
     end
 
+    describe 'windows' do
+      it 'sets the correct family/release for windows ' do
+        platform = scan_with_windows()
```

Comment:
This test tests if the additional check in lib/train/platforms/detect/specifications/os.rb  line 29 can detect windows even without explicit WinRM transport. So, I assume this tests if it works for other transports like SSH.

It was really hard for me to find a way how I can test this, I tried several things in ssh_connection_test.rb, but it didn't matter if I comment out my new code or not. So finally I found this test.